Solutions/BitSight/Analytic Rules/BitSightCompromisedSystemsDetected.yaml

```diff
@@ -0,0 +1,44 @@
+id: d68b758a-b117-4cb8-8e1d-dcab5a4a2f21
+name: BitSight - compromised systems detected
+description: |
+  'Rule helps to detect whenever there is a compromised systems found in BitSight.'
+severity: Medium
+status: Available
+requiredDataConnectors:
+  - connectorId: BitSight
+    dataTypes:
+      - BitSightFindingsData
+queryFrequency: 1d
+queryPeriod: 24h
+triggerOperator: GreaterThan
+triggerThreshold: 0
+tactics:
+  - Execution
+query: |
+    let timeframe = 24h;
+    BitSightFindingsData
+    | where ingestion_time() > ago(timeframe)
+    | where RiskCategory == "Compromised Systems"
```

Comment:
According to the sample data, RiskCategory == "Compromised Systems" will cover "botnet_infections". This will be a duplicate with BigSightBotnetInfectionDetection.
Either remove the BotnetInfectionDetection or make this detection more specific.